### ISSUE TITLE
Corner Resize Size

### ISSUE THREAD

**Issue by WASasquatch** [comfyanonymous/ComfyUI#584]
Is there a way we can *dramatically* increase the corner resize event area? It's so hard to grab the resize area. It's like it's one pixel or something.

**Comment by comfyanonymous** [comment]
Yeah this was bothering me a bit too, it should be fixed now: https://github.com/comfyanonymous/ComfyUI/commit/4cea9aecdab6bbd7b5801c64c27368ee3203a9ad

**Comment by WASasquatch** [comment]
Sweet I'll update and take a look as soon as my modem stops being a twit. Got no upload at all :/
